ui/client/components/HealthCheck.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import {connect} from "react-redux"
+import ActionsUtils, {EspActionsProps} from "../actions/ActionsUtils"
+import InlinedSvgs from "../assets/icons/InlinedSvgs"
+import HttpService from "../http/HttpService"
+import PeriodicallyReloadingComponent from "./PeriodicallyReloadingComponent"
+import {withTranslation} from "react-i18next"
+import {WithTranslation} from "react-i18next/src"
+import {compose} from "redux"
+
+type HealthCheckResponse = {
+  state: string,
+  error: string,
+}
+
+type OwnProps = {}
```

Comment:
Should we really define even empty own props?

Reply:
Hmm. I've changed it.